kolibri/deployment/default/settings/base.py

```diff
@@ -47,8 +47,10 @@
     'kolibri.auth.apps.KolibriAuthConfig',
     'kolibri.content',
     'kolibri.logger',
+    'kolibri.tasks.apps.KolibriTasksConfig',
```

Comment:
It's for when your app has a custom `AppConfig` -- the tasks app has one, since I need to run something once all apps are ready!